src/main/java/org/cryptomator/common/locationpresets/GoogleDriveMacLocationPresetsProvider.java

```diff
@@ -0,0 +1,28 @@
+package org.cryptomator.common.locationpresets;
+
+import org.cryptomator.integrations.common.CheckAvailability;
+import org.cryptomator.integrations.common.OperatingSystem;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Stream;
+
+import static org.cryptomator.integrations.common.OperatingSystem.Value.MAC;
+
+@OperatingSystem(MAC)
+@CheckAvailability
+public final class GoogleDriveMacLocationPresetsProvider implements LocationPresetsProvider {
+
+	private static final Path LOCATION = LocationPresetsProvider.resolveLocation("~/Google Drive/My Drive");
```

Comment:
Does not exist on my machine either.

Reply:
Fixed in https://github.com/cryptomator/cryptomator/pull/2918/commits/1d6edb8373d9598d80cdf21e1de80c0fd1956b80

Reply:
Please re-add `~/Google Drive/My Drive` as an option with a higher priority. If it exists, you can't save anything in `~/Google Drive` because it's a read-only folder.

Are you absolutely sure that this is not the same on Windows? How do they deal with multiple drives? E.g., if you have a personal drive and a business drive.

Reply:
With Google Drive for Mac (v 74.0.3.0), there is no "My Drive":

```
ls -l ~/Google\ Drive 
lrwx------  1 sebastian  staff  76 17 Mai 19:42 /Users/sebastian/Google Drive -> /Users/sebastian/Library/CloudStorage/GoogleDrive-REDACTED
```

Reply:
@tobihagemann do you know someone with mutliple Google Drive accounts? Maybe they can check whethere there are multiple `~/Library/CloudStorage/GoogleDrive-*`, which we can check instead of the symlink.

Reply:
No but I'm certain that it goes something like this:

1. Google Drive supports multiple accounts and then there will be multiple accounts at `~/Library/CloudStorage/GoogleDrive-*`.
2. Google Drive support multiple drive per account and then there will be multiple drives at `~/Library/CloudStorage/GoogleDrive-*/*`.

Not sure how to illustrate it better. 😄 And if we really want to go down this rabbit hole, we have to consider that "My Drive" is localizable, see: https://github.com/cryptomator/cryptomator/issues/1930#issuecomment-970165322

Reply:
Ah yes, we have other test accounts at Google. And yes, there are now multiple folders:

```
/Users/saibot/Library/CloudStorage/GoogleDrive-<redacted-email-1>
/Users/saibot/Library/CloudStorage/GoogleDrive-<redacted-email-2>
```

And both have a `My Drive` subdirectory. Unfortunately, I don't have business accounts for testing purposes. But I'd guess that there will be more subdirectories than `My Drive`.

That's the whole problem that we currently have with the mobile apps that we don't support shared drives. https://developers.google.com/drive/api/guides/about-files

Reply:
@infeo keep `~/Google Drive/My Drive` for this PR. I'll create an issue for further improvements.

Reply:
Changed in https://github.com/cryptomator/cryptomator/pull/2918/commits/179240b325cf2d874bdcd4a969cfff8cfd395c62